build

```diff
@@ -1,29 +1,31 @@
 #!/bin/bash -e
 
+CDIR=$(cd `dirname $0` && pwd)
```

Comment:
Yes, when you run scripts from home directory for example. `cd ~ ; sudo ./fleet./functional/test`